### PR TITLE
dev: [WIP][DO NOT MERGE] redeploying of Cloud Function to modify ENVVAR failed during the CFT test

### PR DESCRIPTION
**Pull request by CienetStingLin**

### CODE REVIEW

File: infra/test/integration/simple_example/simple_example_test.go [hunk 1]
```diff
@@ -96,6 +96,11 @@ func TestSimpleExample(t *testing.T) {
 			return false, nil
 		}
 		utils.Poll(t, isServing, 20, time.Second * 3)
+
+		// Update annotate_gcs env vars
+		sourceCodeUrl := example.GetStringOutput("source_code_url")
+		gcloud.Run(t, fmt.Sprintf("functions deploy %s --region=%s --trigger-bucket=%s --source='%s' --gen2 --update-env-vars='FEATURES=TEXT_DETECTION' --format=json",
```
Comment (bharathkkb) [inline infra/test/integration/simple_example/simple_example_test.go:102]: Looks like this was causing some weird escaping issue
```suggestion
		gcloud.Run(t, fmt.Sprintf("functions deploy %s --region=%s --trigger-bucket=%s --source=%s --gen2 --update-env-vars='FEATURES=TEXT_DETECTION' --format=json",
```

Comment (CienetStingLin) [reply infra/test/integration/simple_example/simple_example_test.go]: Thank you for your assistance.

**Pull request closed without merging**